### PR TITLE
[ElasticSearch] increase default timeout

### PR DESCRIPTION
**Pull request by hackathon12344**
## What
Original github source code had issue with elastic search connector timeout after 30k milliseconds. 

## How
This PR increases this default timeout at clientBuilder to 30k*100k millisecs.

## Review guide
the clientbuilder in the Elastic search connection builder sets this default value and this pr increases this timeout. 

## User Impact
Longer timeout for elastic search scrolls in the default client.

## Can this PR be safely reverted and rolled back?
- [x ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by evantahler** [comment]
That's a really long timeout... like 34 days.  That's probably too long.

**Review by evantahler — COMMENTED** [review]
This change also only applies to the connection when a custom SSL certificate is in use.  I would imagine the change should be applied to the `clientBuilder` in all cases?

**Review by evantahler — CHANGES_REQUESTED** [review]
Suggestions for improvement:
* apply the `clientBuilder.setSocketTimeout` changes such that they apply regardless of if a custom SSL certificate is in use
* rather than choose a single large value for the timeout, use the connector's SPEC options to make this configurable, and start with a much smaller default (perhaps 30min, which is the default Elasticsearch chose).  The code will need to be updated to use this value from the configuration passed to the connector at boot
* Update the connector's version, metadata, and docs, so that a new version can be published

**Comment by hackathon12344** [comment]
> That's a really long timeout... like 34 days. That's probably too long.

The timeout for SSL was specific to my local environment hence I put there. But I see your point and have moved it outside this block. I have reduced it to 1 hour. The default 30 seconds was not enough for elastic search scroll records to scroll through large results of data. I am not sure if this 1 hour will work either. Hence, I put a random large number of 1 day and 34 days before it. I think we can test it and see. 

Spec options - That makes sense to pass this as a configurable value. Let me check the spec file on how to achieve this.

**Comment by hackathon12344** [comment]
>     * Update the connector's version, metadata, and docs, so that a new version can be published

Thanks for the feedback.
In metadata yaml:
metadataSpecVersion: "1.0" has to be changed to "1.1" ?
or   dockerImageTag: 0.1.6 ? to 0.1.7 ?

Where is the connector version specified ?

I see the resources/spec.json. But I could not see an example where this resource gets used in the main src class files.

**Comment by evantahler** [comment]
Versions: Both the metadata.yaml and docker image tags should match.
Here's an example PR (https://github.com/airbytehq/airbyte/pull/35304/files) of how the java code loads an option set in the spec.  This PR added a new option to the spec invalid_cdc_cursor_position_behavior and then references it later via `sourceConfig.get("replication_method")`

File: airbyte-integrations/connectors/source-elasticsearch/src/main/java/io/airbyte/integrations/source/elasticsearch/ElasticsearchConnection.java [hunk 1]
```diff
@@ -53,7 +53,10 @@ public ElasticsearchConnection(ConnectorConfiguration config) {
 
     HttpHost httpHost = HttpHost.create(config.getEndpoint());
 
-    RestClientBuilder builder = RestClient.builder(httpHost).setDefaultHeaders(configureHeaders(config)).setFailureListener((new FailureListener()));
+    RestClientBuilder builder = RestClient.builder(httpHost)
+            .setDefaultHeaders(configureHeaders(config))
+            .setSocketTimeout(3600000);//increase from default 30,000 millisecs to 1 hour
+            .setFailureListener((new FailureListener()));
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-elasticsearch/src/main/java/io/airbyte/integrations/source/elasticsearch/ElasticsearchConnection.java:59]: Can you check this line pelase?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**